docs/team/member-guide.md

```diff
@@ -0,0 +1,96 @@
+# JupyterHub team guide
+      
+This page holds resources for members of the JupyterHub and Binder teams.
+They're meant to guide team members to be happy, productive members of the
+team!
+
+## What are the team resources?
+   
+There are a few resources that are particularly useful for team members. Here's
+a quick list to get you started.
+   
+* [**The JupyterHub Team Compass**](https://jupyterhub-team-compass.readthedocs.io/en/latest/contributing.html)
+  is a repository with lots of information about team-related things. It has
+  development tips, information about team meetings, milestones and roadmaps,
+  etc.
+* [**The JupyterHub Team Compass issues**](https://github.com/jupyterhub/team-compass/issues)
+  are where we often discuss specific, actionable things related to the *team*
+  (e.g., discussing whether to change something in the team-compass repo).
+* [**The JupyterHub internal Discourse category**](https://discourse.jupyter.org/c/jupyterhub/internal)
```

Comment:
I'm not a huge fan of having a not-public place for discussions. What should go there that can't be in one of the GitHub repos, a public discourse thread, gitter or the private mailing list (we already have and use sparingly)? So I would remove it from the guide as well as the forum.

From my side there doesn't seem to be a need for a private place for discussions. However this is a biased view based on not seeing discussions that should have been private (but were done in public) nor feeling that there are things I'd want to discuss but can't because we lack the venue. The bias here is that if others would have started conversations but didn't because we lack the place for it I would not notice :)

Reply:
Right now I'm just trying to describe what's already there - I'm happy to discuss removing the Discourse (and more generally, where to have private conversations) as a group, but could we do it in a different issue instead of in this PR?

Reply:
Think about it some more - I'm gonna remove the sections that talk about Discourse because I don't think we really have a team processes worked out around this yet.

I think we've discussed moving more general team discussion from `team-compass` issues to the Discourse, but not sure where's the best place for that.